oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -1066,13 +1068,21 @@ public void failed(Exception e) {
                 // reusing value so value display is consistent
                 failingSince = latestErrorTime;
                 latestErrorWarn = System.currentTimeMillis();
-                log.warn("[{}] The index update failed", name, e);
+                if (isConcurrentUpdateException) {
+                    log.info("[{}]", name,  e.getMessage());
+                } else {
+                    log.warn("[{}] The index update failed", name, e);
+                }
             } else {
                 // subsequent occurrences
                 boolean warn = System.currentTimeMillis() - latestErrorWarn > ERROR_WARN_INTERVAL;
                 if (warn) {
-                    latestErrorWarn = System.currentTimeMillis();
```

Comment:
I would keep latestErrorWarn where it is... same as you did above.

Reply:
done